File: WebFt30/src/app/core/store/models/monitoring-baseline.ts
import { MeasurementSettings } from './measurement-settings';

export class MonitoringBaseline {
  public id!: number;
  public monitoringPortId!: number;
  public createdByUserId!: string;
  public createdAt!: Date;
  public measurementSettings!: MeasurementSettings;
}

File: Backend/src/Fibertest30.Application/Monitoring/Models/MonitoringBaseline.cs
namespace Fibertest30.Application;

public class MonitoringBaseline
{
    public int Id { get; init; }
    
    public int MonitoringPortId { get; set; }
    public string CreatedByUserId { get; init; } = string.Empty;
    public DateTime CreatedAt { get; init; }
    public MeasurementSettings MeasurementSettings { get; init; } = null!;
}